integrated_channels/sap_success_factors/migrations/0011_alter_sapsuccessfactorslearnerdatatransmissionaudit_index_together.py

```diff
@@ -3,6 +3,20 @@
 from django.db import connection, migrations
 
 
+def generate_index_sql(db_engine):
+    if 'mysql' in db_engine:
+        return """
+                        CREATE INDEX sapsf_sldta_85936b55_idx
+                        ON sap_success_factors_sapsuccessfactorslearnerdatatransmission3ce5 (enterprise_customer_uuid, plugin_configuration_id)
+                        ALGORITHM=INPLACE LOCK=NONE
+                    """
+    elif 'postgresql' in db_engine:
+        return """
+                        CREATE INDEX sapsf_sldta_85936b55_idx
+                        ON sap_success_factors_sapsuccessfactorslearnerdatatransmission3ce5 (enterprise_customer_uuid, plugin_configuration_id)
+                    """
```

Comment:
NIt: I think this is okay either way, but if it's only being used once, I think you might as well inline it like you did elsewhere in this PR.

Reply:
noted